src/Controller/GameController.php

```diff
@@ -17,4 +19,19 @@ public function play()
     {
         return $this->twig->render('Game/play.html.twig');
     }
+
+    public function battle()
+    {
+        $stercketUser = new Stercket();
+        $stercketEnnemy = new Stercket();
+        $stercketUser->initialise('player');
+        $stercketEnnemy->initialise('wood');
+        return $this->twig->render('Game/battle.html.twig', [
+            "stercketUser" => $stercketUser,
+            "healthStercketUser" => $stercketUser->getHealth(),
```

Comment:
You already send the stercketUser to twig, use it to access it's health directly in twig

Reply:
For the moment we need to keep an acces to the stercket health before the combat to test  a futur status method which will define a state for the stercket concerning its hp